backend/src/SessionRepository.js

```diff
@@ -0,0 +1,26 @@
+class SessionRepository {
+	constructor( driver ) {
+		this._driver = driver;
+	}
+
+	async getAll( issueKey ) {
+		const issueSessions = {};
+		const response = await this._driver.client.hgetall( issueKey );
+
+		for ( const session in response ) {
+			issueSessions[ session ] = JSON.parse( response[ session ] );
+		}
+
+		return issueSessions;
+	}
+
+	async createOrUpdate( issueKey, socketId, issueSession ) {
```

Comment:
maybe just `set`?

Reply:
Sure, it's simpler. I wanted to emphasize that this function can both create and update existing data depend on data existed before. But `set` is descriptive enough.